targets/TARGET_STM/TARGET_STM32L4/TARGET_MTS_DRAGONFLY_L471QG/PinNames.h

```diff
@@ -0,0 +1,399 @@
+	D5  = PA_9,
+	D6  = PA_1,
+	D7  = PG_7,
+	D8  = PB_0,
+	D9  = PB_10,
+	D10 = PC_8,
+	D11 = PB_5,
+	D12 = PG_3,
+	D13 = PG_2,
+	D14 = PB_9,
+	D15 = PB_8,
+
+	// 40 pin
+	IO_00 = D1,
+	IO_01 = D4,
+	IO_02 = D8,
+	IO_03 = D6,
+	IO_04 = D11,
+	IO_05 = D13,
+	IO_06 = D15,
+	IO_07 = D2,
+	IO_08 = A0,
+	IO_09 = A3,
+	IO_10 = A1,
+	IO_11 = A4,
+	IO_12 = A2,
+	IO_13 = D9,
+	IO_14 = A5,
+	IO_15 = D14,
+	IO_16 = D12,
+	IO_17 = D10,
+	IO_18 = D3,
+	IO_19 = D5,
+	IO_20 = D7,
+	IO_21 = D0,
+
+	// 40 pin USB debug port
+	USBTX = PB_6,
+	USBRX = PB_7,
+
+	//UARTS
+	UART3_TX = PD_2,
+	UART3_RX = PD_9,
+	UART3_CTS = PD_11,
+	UART3_RTS = PD_12,
+
+	UART2_TX = PA_2,
+	UART2_RX = PA_3,
+	UART2_RTS = PA_1,
+	UART2_CTS = PA_0,
+	UART2_DSR = PA_9,
+	UART2_DTR = PG_7,
+	UART2_DCD = PA_7,
+
+
+	UART1_TX = PB_6,
+	UART1_RX = PB_7,
+	UART1_RTS = PA_12,
+	UART1_CTS = PA_11,
+
+	// 40 pin JTAG/SWD
+	J_TCK = PA_14,	// a.k.a. SWCLK
+	J_TDI = PA_15,
+	J_RST = PB_4,
+	J_TDO = PB_3,	// a.k.a. SWO
+	J_TMS = PA_13,	// a.k.a. SWDIO
+
+	// Generic signals namings
+	LED1 = D3,
+	LED2 = D3,
+	LED3 = D3,
+	LED4 = D3,
+
+	// SERIAL PORT
+    SERIAL_TX  = D1,
+    SERIAL_RX  = D0,
+    SERIAL_RTS = D6,
+    SERIAL_CTS = D3,
+    SERIAL_DCD = D4,
+    SERIAL_DSR = D5,
+    SERIAL_DTR = D7,
+    SERIAL_RI  = D8,
+
+	// SPI1 and SPI2 are available on Arduino pins
+    SPI1_MOSI = D11,
+    SPI1_MISO = D12,
+    SPI1_SCK  = D13,
+    SPI2_MOSI = D2,
+    SPI2_MISO = A0,
+    SPI2_SCK  = D9,
+
+    // SPI3 connects to flash part
+	SPI3_MOSI = PC_12,
+	SPI3_MISO = PC_11,
+	SPI3_SCK  = PC_10,
+	SPI3_SSEL = PG_12,
+
+    // I2C1 and I2C3 are available on Arduino pins
+    I2C1_SCL = D15,
+    I2C1_SDA = D14,
+    I2C3_SCL = D7,
+    I2C3_SDA = A5,
+
+	// Modem
+	RADIO_PWR   = PC_3,
+	RADIO_RESET = PF_3,
+	RADIO_TX    = PD_8,
+	RADIO_RX    = PD_9,
+	RADIO_RI    = PD_10,
+	RADIO_CTS   = PD_11,
+	RADIO_RTS   = PD_12,
+	RADIO_DTR   = PD_13,
+	RADIO_DCD   = PD_14,
+	RADIO_DSR   = PD_15,
+	RADIO_ONOFF = PE_4,     //rev b PC_13; rev c PC_0 PE_4
+	MON_1V8     = PC_5,
+	VUSB_EN		= PE_3,
+	// Power control for level shifter and SPI flash. Low powers on.
+	BUF_EN	= PC_6,
+	VMEM_EN	= PE_1,
+	VUSB_DET = PE_3,
+
+	MDMPWRON = RADIO_ONOFF, // 3G_ONOFF DragonFly Design Guide, Page No. 16
+	MDMTXD   = RADIO_TX, // Transmit Data
+	MDMRXD   = RADIO_RX, // Receive Data
+	MDMRTS   = RADIO_RTS, // Request to Send
+	MDMCTS   = RADIO_CTS, // Clear to Send
+	MDMDCD   = RADIO_DCD, // Data Carrier Detect
+	MDMDSR   = RADIO_DSR, // Data Set Ready
+	MDMDTR   = RADIO_DTR, // Data Terminal Ready
+	MDMRI    = RADIO_RI, // Ring Indicator
+
+
+	CELL_GPIO5 = PE_11,
+	CELL_GPIO2 = PE_13,
+	CELL_GPIO3 = PE_14,
+	CELL_GPIO4 = PE_15,
+
+	// Voltage measuring
+	VCC_IN_MEAS_EN = PB_12,
+	VCC_IN_MEAS = PC_0,
+
+	// added to support ppp
+	MDM_PIN_POLARITY = 0,
+	MDMRST = RADIO_RESET,
+	MDMCURRENTSENSE = (int) 0xFFFFFFFF,
+
+    // GNSS
+    GNSSEN  = (int) 0xFFFFFFFF,     // VCC_IO to GNSS, should be set to push-pull, no pull-up, output
```

Comment:
Can you fix alignment, also in the file targets.json

Reply:
No problem